devicon.json

```
@@ -1095,20 +1095,23 @@
         "altnames": [],
         "tags": [
             "jetbrains",
-            "editor"
+            "editor",
+            "c",
+            "c++",
+            "cpp",
+            "cplusplus"
         ],
         "versions": {
             "svg": [
                 "original",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
```

I don't see a reason why these should be removed
```suggestion
                "original-wordmark",
                "plain",
                "plain-wordmark"
```

---

@Snailedlt It's normal, @kilian-paquier is correct to remove `plain` from the SVG list, `plain` exists normally only for the font, so `plain` SVG need to be referenced only on the `"font": [` list.

---

To nuance my previous message its seems that sometime `plain` versions are also listed in the `"svg"` list, like for example `heroku` or `haskell`.

---

@Snailedlt I removed my previous comments , i was wrong and yes your edit request is valid, my bad 🙂

---

@BenSouchet I didn't see it, and no worries :)